katdal/chunkstore.py

```diff
@@ -386,3 +432,38 @@ def put_dask_array(self, array_name, array, offset=()):
         # The success array has one element per chunk in the input array
         out_chunks = tuple(len(c) * (1,) for c in array.chunks)
         return da.Array(dask_graph, out_name, out_chunks, np.object)
+
+    def has_dask_array(self, array_name, chunks, dtype, offset=()):
+        """Check if dask array is in the store.
+
+        Parameters
+        ----------
+        array_name : string
+            Identifier of array in chunk store
+        chunks : tuple of tuples of ints
+            Chunk specification
+        dtype : :class:`numpy.dtype` object or equivalent
+            Data type of array
+        offset : tuple of int, optional
+            Offset to add to each dimension when addressing chunks in store
+
+        Returns
+        -------
+        success : :class:`dask.array.Array` object
+            Dask array of bools indicating presence of each chunk
+
+        Raises
+        ------
+        :exc:`chunkstore.StoreUnavailable`
+            If interaction with chunk store failed (offline, bad auth, bad config)
```

Comment:
Does it actually raise it, or do you get back a graph that raises it when it tries to compute?

Reply:
The latter. Not sure how to indicate this...

Reply:
I would just write about it in the general description, instead of having a section.

Reply:
Done. I'm leaving this out until I figure out good error handling for this part anyway.